__tests__/selenium_tests/e2e_tests.ts

```diff
@@ -187,7 +351,7 @@ describe('End to end test', () => {
 
       logger.info('sending deployment information')
       const deploymentData = await fs.readFile(deploymentMetadataPath, { encoding: 'utf-8' })
-      const deploymentInfo = JSON.parse(deploymentData)
+      const deploymentInfo = { ...JSON.parse(deploymentData) }
```

Comment:
What's the purpose of this expansion?

Reply:
I think I changed this to something else then changed it back! Will revert.